HW_15/homework.py

```diff
@@ -0,0 +1,69 @@
+# Напишите код, который запускается из командной строки и получает на вход путь до директории на ПК.
+# Соберите информацию о содержимом в виде объектов namedtuple.
+# Каждый объект хранит:
+# * имя файла без расширения или название каталога,
+# * расширение, если это файл,
+# * флаг каталога,
+# * название родительского каталога.
+# В процессе сбора сохраните данные в текстовый файл используя логирование.
+
+import os
+import os.path
+from pathlib import Path
+import logging
+from collections import namedtuple
+import argparse
+
+logging.basicConfig(filename="homework.log",
+                    encoding="utf-8",
+                    level=logging.DEBUG)
```

Comment:
Вы же используете только `info` регистратор. Уровень `DEBUG` не нужен
```suggestion
                    level=logging.INFO)
```